### DISCUSSION TITLE
部署架构 - DaoCloud Enterprise

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#979] [Announcements]
# 部署架构 - DaoCloud Enterprise

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/install/commercial/deploy-arch/

**Comment by zhoujw1974** [comment]
不同部署方式的应用场景最好也说明一下，写得太简单了

**Reply by Michelle951** [reply]
@jzhupup

**Comment by windsonsea** [comment]
可以参考首页的 9 大场景：https://docs.daocloud.io/dce/what/
